README.md

```diff
@@ -83,7 +83,7 @@ Setting up the credentials can be done as follows:
 
```

Comment:
**unregister_on_halt** disables or enables automatic unregistration on halt (on shut down). By default the plugin unregisters on halt, you can however change that by setting the option to `false` so that the box will unregister only on destroy.

**manager** selects the registration manager provider. By default the plugin uses `subscription_manager` as the registration manager provider, you can, however, change that by setting the option to a different manager: